controllers/cloud.redhat.com/clowdjobinvocation_controller.go

```diff
@@ -170,55 +168,57 @@ func (r *ClowdJobInvocationReconciler) Reconcile(ctx context.Context, req ctrl.R
 	var emptyTesting crd.IqeJobSpec
 	if cji.Spec.Testing.Iqe != emptyTesting {
 
+		randomString := utils.RandStringLower(7)
 		nn := types.NamespacedName{
-			Name:      fmt.Sprintf("%s-iqe", cji.Name),
+			Name:      fmt.Sprintf("%s-iqe-%s", cji.Name, randomString),
```

Comment:
We could have another method on the CJI to generate a name, I would specifically call it "generate" so that people know you can't just recall it with a call to the function